deployments/mainnet/deployment.json

```diff
@@ -0,0 +1,108 @@
+{
+  "beamer_commit": "274eee899af5cff78dc1a682240d4832a7eff8d5",
+  "deployer": "0x42405d66fdA09dbDaC90Ff25fC5a4C2353f43E70",
+  "L1": {
+    "Resolver": {
+      "address": "0x5193a6309FF3e72855dcC0c9d5C51fBBD2439c32",
+      "deployment_block": 15082748,
+      "deployment_args": []
+    },
+    "OptimismL1Messenger": {
+      "address": "0x913b5aC7E1124Ea3429593193bf44eB967FB7658",
+      "deployment_block": 15082755,
+      "deployment_args": [
+        "0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1"
+      ]
+    }
+  },
+  "L2": {
+    "288": {
+      "MintableToken": {
```

Comment:
I think there shouldn't be a mintable token for mainnet, even if it was deployed

Reply:
> I think there shouldn't be a mintable token for mainnet, even if it was deployed

Does it harm? Actually, it was not deployed the second time, we're reusing the 1st deployment that you did.
The Rinkeby one has ti too and it makes testing easier because we don't have to track the addresses manually.

However, I can drop that commit if you still think it should not be there.

Reply:
After discussion, please remove it from the mainnet deployment information as we will use actual tokens with value for testing

Reply:
Done.